main.py

```diff
@@ -1,39 +1,58 @@
+# -*- encoding: utf-8 -*-
 import argparse
-from converter.converter import *
+from typing import Any
 
+from converter.converter import *
 
 CELSIUM_INPUT = ['celsius', 'cels', 'cel', '-c', 'c']
 FAH_INPUT = ['fahrenheit', 'fahren', 'far', '-f', 'f']
 QUIT_INPUT = ['quit', 'q', 'exit']
 
-
 parser = argparse.ArgumentParser()
 parser.add_argument("-v", "--verbose", help="It just return your argument")
 parser.add_argument("-f", "--fahrenheit", help="Take temperature in Fahrenheit and convert to Celsius", type=float)
 parser.add_argument("-c", "--celsius", help="Take temperature in Celsius and convert to Fahrenheit", type=float)
 
 args = parser.parse_args()
 
+
+def is_float(arg: Any) -> bool:
```

Comment:
@MariaMaximova, I would suggest moving this function to separate file, e.g. `utils.py`. What do you think?

Reply:
@OleksiiBulba [Some fixes. Branch issue-5](https://github.com/MariaMaximova/convert_temp/pull/9/commits/5552c66100a2c6836c6083729ca28eda94e9211e)